.github/workflows/build-&-publish-docker-image.yml

```diff
@@ -53,15 +71,33 @@ jobs:
           docker tag blobber:latest ${BLOBBER_REGISTRY}:$TAG-$SHORT_SHA
           docker push ${BLOBBER_REGISTRY}:$TAG-$SHORT_SHA  || { sleep 10 && docker push ${BLOBBER_REGISTRY}:$TAG-$SHORT_SHA; }
 
-  build_and_publish_validator:
-    runs-on: [self-hosted, build]
+  validator:
+    runs-on: [self-hosted, docker-builds]
```

Comment:
what's the difference between build and docker-builds? looks like the same rulers are tagged with both?

Reply:
yes. they are same on https://github.com/0chain/blobber/settings/actions/runners now